app/src/main/java/com/programmers/calculator/repository/HistoryRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.programmers.calculator.repository;
+
+import com.programmers.calculator.model.Expression;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HistoryRepository{
+    List<String> history;
+    List<Integer> resultHistory;
```

Comment:
```suggestion
public class HistoryRepository{
    private final List<String> history;
    private final List<Integer> resultHistory;
```

expression과 결과를 같이 저장하는건가요? 그럼 새 객체(expression, result를 포함한)를 만들어서 저장하는게 좋아보이네요.